api/src/services/platform-service.ts

```diff
@@ -70,6 +71,12 @@ export interface IArtifact {
   };
 }
 
+interface publishIds {
```

Comment:
Nit: Maybe rename this interface to `IPublishRecord`?

Reply:
As it stands, we're using `publishIds` as both a variable name and a type, which could be confusing